packages/public/server/__tests-e2e__/home.ts

```diff
@@ -21,8 +21,8 @@
  */
 import { Browser, launch, Page } from 'puppeteer'
 
-const seconds = 1000
-jest.setTimeout(60 * seconds)
```

Comment:
This was intended. The second line should be read as: "the timeout is 60 seconds"

Reply:
I read the line as "Timeout is 60 seconds" because the variable was named `seconds`. Then I would rename the variable as `milisecondsPerSecond` or I would write `jest.setTimeout(60 * 1000)`...

Reply:
> I read the line as "Timeout is 60 seconds" 
Yep, that is what I wrote? :) Or did you mean "Timeout is 1000 seconds"?

If so and you don't accept my explanation (😅), you could move that into a function to make it clearer, e.g.
```
setTimeout(60)

function setTimeout(seconds: number) {
  jest.setTimeout(seconds * 1000)
}
```
which would IMHO combine both of our understandings of the two approaches we seen.

Reply:
Okay, I moved it to a global `jest.setup-e2e.js` file in https://github.com/serlo/serlo.org/blob/b0dffdc027016b4798d32dc7549402c855c423ee/jest.setup-e2e.js#L1-L5

Reply:
Awesome, thanks